src/StreamMaster.h

```diff
@@ -182,7 +182,7 @@ template <typename Streamer> class StreamMaster {
       } catch (std::exception &E) {
         LOG(Sev::Error, "Stream closed due to stream error: {}", E.what());
         logEvent(ProducerTopic, StatusCode::Error, ServiceId,
-                 WriterTask->job_id(), E.what());
+                 WriterTask->jobID(), E.what());
         closeStream(Stream, Demux.topic());
         return StreamMasterError::STREAMER_ERROR();
```

Comment:
I would like the class `StreamMasterError` (in its current version) to die in a fire but I guess it will have to wait